src/main/java/qna/domain/Question.java

```diff
@@ -53,9 +60,21 @@ public boolean isOwner(User writer) {
     }
 
     public void addAnswer(Answer answer) {
+        if (answers.contains(answer)) {
+            return;
+        }
+        answers.add(answer);
         answer.toQuestion(this);
     }
```

Comment:
```suggestion
    public void addAnswer(Answer answer) {
        answers.add(answer);
    }
```

이렇게만 되어도 충분하지 않을까요? 나머지 코드는 `Answers`의 책임이 아닐지 생각이 들어요 :thinking:

Reply:
좋은 의견 감사합니다! 
이미 컬랙션에 존재하는 answer인지 확인하는건 Answers가 해야되는 역할이 맞겠네요!